src/task_types/burn_in_tests.rs

```diff
@@ -231,15 +332,14 @@ impl BurnInServiceImpl {
         &self,
         params: &ResmokeGenParams,
         index: usize,
-        test: &str,
         suite_info: &BurnInSuiteInfo,
-    ) -> EvgTask {
+    ) -> GeneratedSubTask {
         let origin_suite = suite_info.build_origin_suite(&params.suite_name);
 
         let sub_suite = SubSuite {
             index,
             name: suite_info.build_display_name(),
-            test_list: vec![test.to_string()],
+            test_list: vec![],
```

Comment:
Don't we need to know the test name here? Will this break burn_in tests?

Reply:
We don't need it here. We use resmoke invocation to pass in the test name [here](https://github.com/mongodb/mongo-task-generator/blob/2f23b6ff9b849763476baac9b4207a31c2498962/src/task_types/burn_in_tests.rs#L395).

As for this `test_list`, it is needed to write a physical suite yaml file. We use it during regular task splitting and never used it for burn_in_tests.